testing/execution_engine_integration/src/genesis_json.rs

```diff
@@ -32,11 +32,102 @@ pub fn geth_genesis_json() -> Value {
+                  "blockReward": {
+                    "0x0": "0x1BC16D674EC80000"
+                  },
+                  "homesteadTransition": "0x0",
+                  "eip100bTransition": "0x0",
+                  "difficultyBombDelays": {}
+                }
+              }
+            },
+            "params": {
+              "gasLimitBoundDivisor": "0x400",
+              "registrar": "0x0000000000000000000000000000000000000000",
+              "accountStartNonce": "0x0",
+              "maximumExtraDataSize": "0x20",
+              "minGasLimit": "0x1388",
+              "networkID": "0x1469ca",
+              "MergeForkIdTransition": "0x3e8",
+              "eip150Transition": "0x0",
+              "eip158Transition": "0x0",
+              "eip160Transition": "0x0",
+              "eip161abcTransition": "0x0",
+              "eip161dTransition": "0x0",
+              "eip155Transition": "0x0",
+              "eip140Transition": "0x0",
+              "eip211Transition": "0x0",
+              "eip214Transition": "0x0",
+              "eip658Transition": "0x0",
+              "eip145Transition": "0x0",
+              "eip1014Transition": "0x0",
+              "eip1052Transition": "0x0",
+              "eip1283Transition": "0x0",
+              "eip1283DisableTransition": "0x0",
+              "eip152Transition": "0x0",
+              "eip1108Transition": "0x0",
+              "eip1344Transition": "0x0",
+              "eip1884Transition": "0x0",
+              "eip2028Transition": "0x0",
+              "eip2200Transition": "0x0",
+              "eip2565Transition": "0x0",
+              "eip2929Transition": "0x0",
+              "eip2930Transition": "0x0",
+              "eip1559Transition": "0x0",
+              "eip3198Transition": "0x0",
+              "eip3529Transition": "0x0",
+              "eip3541Transition": "0x0"
+            },
+            "genesis": {
+              "seal": {
+                "ethereum": {
+                  "nonce": "0x1234",
+                  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+                }
+              },
+              "difficulty": "0x01",
+              "author": "0x0000000000000000000000000000000000000000",
+              "timestamp": "0x0",
+              "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+              "extraData": "",
+              "gasLimit": "0x1C9C380"
+            },
+            "accounts": {
+              "0x7b8C3a386C0eea54693fFB0DA17373ffC9228139": {
+                "balance": "10000000000000000000000000"
+              },
+              "0xdA2DD7560DB7e212B945fC72cEB54B7D8C886D77": {
+                "balance": "10000000000000000000000000"
+              },
+            },
+            "nodes": [
+              "enode://c354db99124f0faf677ff0e75c3cbbd568b2febc186af664e0c51ac435609badedc67a18a63adb64dacc1780a28dcefebfc29b83fd1a3f4aa3c0eb161364cf94@164.92.130.5:30303"
```

Comment:
Should we remove this so we're not pinging some random IP from CI?

Reply:
Nice catch!